src/Main.java

```diff
@@ -1,5 +1,86 @@
-public class Main {
+import java.time.LocalDate;
+import java.util.Scanner;
+
+class Main {
+    static Scanner sc = new Scanner(System.in);
+
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+        checkleapYear();
+        System.out.println("Задача 2");
+        printInfoAboutVersion();
+        System.out.println("Задача 3");
+        int deliveryDays = calculateDeliveryTime();
+        if (deliveryDays > 0) {
+            System.out.println("Потребуется дней: " + deliveryDays);
+        } else {
+            System.out.println("Доставки нет");
+        }
+
+    }
+
+    //Задача 1
+    public static boolean leapYear(int year) {
```

Comment:
```suggestion
    public static boolean isLeapYear(int year) {
```
методы, которые возвращают boolean, принято начинать с is